src/Microsoft.Extensions.PlatformAbstractions/RuntimeEnvironmentExtensions.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace Microsoft.Extensions.PlatformAbstractions
+{
+    public static class RuntimeEnvironmentExtensions
+    {
+        public static string GetRuntimeIdentifier(this IRuntimeEnvironment self)
```

Comment:
Are we going to support/respect (if already supported in the cli) an equivalent of the `DNX_RUNTIME_ID` env variable? It does not seem the code currently supports this.